app/index.html

```diff
@@ -52,7 +52,7 @@
     <![endif]-->
       <header>
         <section class="nav">
-            <a href="http://hood.ie" class="logo"><img src="images/logo.svg" width="135" alt=""></a>
+            <a href="http://hood.ie" class="logo"><img src="images/logo.svg" width="135" alt=""><b>Hoodie</b></a>
```

Comment:
maybe it would have the same effect if we'd set `alt="Hoodie"`? I'm curious, could you test that with the screen readers?